<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="maximumSize">
             <size>
              <width>105</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Таблица. Вариант4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="maximumSize">
             <size>
              <width>110</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="rowCount">
             <number>10</number>
            </property>
            <property name="columnCount">
             <number>2</number>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>43</number>
            </attribute>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <row/>
            <column/>
            <column/>
            <item row="0" column="0">
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item row="1" column="1">
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item row="2" column="0">
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item row="2" column="1">
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item row="3" column="0">
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item row="3" column="1">
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item row="4" column="0">
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item row="4" column="1">
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item row="5" column="0">
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item row="5" column="1">
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item row="6" column="0">
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item row="6" column="1">
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item row="7" column="0">
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item row="7" column="1">
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item row="8" column="0">
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item row="8" column="1">
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item row="9" column="0">
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item row="9" column="1">
             <property name="text">
              <string>6</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>3</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>минимакс</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QTextBrowser" name="textBrowser">
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>произведений</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTextBrowser" name="textBrowser_2"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_3">
           <attribute name="title">
            <string>Граф.(минимакс)</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCustomPlot" name="customPlot" native="true">
              <property name="minimumSize">
               <size>
                <width>500</width>
                <height>500</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="sizeConstraint">
               <enum>QLayout::SetMaximumSize</enum>
              </property>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Оптимальное решение:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_2"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_4">
           <attribute name="title">
            <string>Граф. (произведений)</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QCustomPlot" name="customPlot_2" native="true"/>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Оптимальное решение:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>решить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
